signing/signing.go

```diff
@@ -0,0 +1,105 @@
+package signing
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/ossf/scorecard-action/entrypoint"
+	"github.com/ossf/scorecard-action/options"
+	sigOpts "github.com/sigstore/cosign/cmd/cosign/cli/options"
+	"github.com/sigstore/cosign/cmd/cosign/cli/sign"
+)
+
+func SignScorecardResult(scorecardResultsFile string) error {
```

Comment:
does this need to be public or can it be kept private?

Reply:
changed to private

Reply:
ended up changing it back to public because it made more sense to call this from main.go than from ProcessSignature in signing.go